components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/IdentityManagementEndpointUtil.java

```diff
@@ -692,4 +692,24 @@ public static String getBasePath(String tenantDomain, String context, boolean is
         }
         return basePath;
     }
+
+    /**
+     * Get a query parameter value from a URL.
+     *
+     * @param url               URL
+     * @param queryParameter    Required query parameter name
+     * @return Query parameter value
+     * @throws URISyntaxException
```

Comment:
Shall we make this comment formatted like the other comments. Like having full stops at the EOL, having exception explanation comment. WDYT?

Reply:
Addressed in https://github.com/wso2/carbon-identity-framework/pull/3171/commits/09b57785a1049994143d57f2e0620c3f88376e14